qemu/tests/block_copy.py

```diff
@@ -499,3 +499,46 @@ def verify_md5s(self):
         file_names = self.params["file_names"].split()
         for name in file_names:
             self.verify_md5(name)
+
+    def mount_disks_linux(self):
```

Comment:
1. Based on you current implantation, the function should named `mount_disks`, it might be good to print a log when `os_type = windows"`, so everything is presented explicitly.

2. if you decide to name it  `mount_disks_linux`(which I think should be `mount_linux_disks`, to be consistent with `format_linux_disk`)， I don't think you
need the if clause inside the method.

Reply:
@hereischen I will prefer `mount_disks()`, since I have to keep the `if` adjustment to allow user call it without bothering guest os distinguish and make sure not fail windows, however, to print log for windows looks wired since for sure windows do not need mount.
So would you accept me only removing `linux` from the function name but without printing log for windows?
Always it is okay for me if you insist a log printing, would you help indicate the log content if you do need? Thanks.

Reply:
OK for me